### PR TITLE
UF-305 - Upgrade GwtBootstrap to version 0.9.3

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -761,6 +762,17 @@
...
       <dependency>
+        <groupId>org.gwtbootstrap3</groupId>
+        <artifactId>gwtbootstrap3</artifactId>
+        <version>${version.org.gwtbootstrap3}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.gwtbootstrap3</groupId>
+        <artifactId>gwtbootstrap3-extras</artifactId>
+        <version>${version.org.gwtbootstrap3}</version>
+      </dependency>
```
Comment (psiroky) [inline pom.xml]: This should not be needed anymore. Overwriting the version property is enough.

**Comment by psiroky** [comment]
@cristianonicolai could you please also create a PR for ip-bom? That way we can have the version centralized and don't need to override it in Errai, UF, Dashbuilder and KIE repos separetly. Thanks!

https://github.com/jboss-integration/jboss-integration-platform-bom/blob/master/pom.xml#L241

**Comment by cristianonicolai** [comment]
@psiroky I created the PR for ip-bom https://github.com/jboss-integration/jboss-integration-platform-bom/pull/326, I guess we need a new release (CR5?) so I can update my PRs.

**Comment by psiroky** [comment]
@cristianonicolai thanks for the IP BOM PR. It's fine to temporarily override the version in UF (+Dash,+KIE) POM (using just the version property). Once the IP BOM is upgraded, then we can remove that property. Sorry if I was not clear before -- please do use the temporary override.

There are certain processes around the IP BOM and it usually takes a few days before the changes get acked and we can release new version. So in order to not stale this PR, it's fine to use the temporary override.

**Comment by cristianonicolai** [comment]
@psiroky got it. I've updated the related PRs to use the property override only, thanks.

File: uberfire-extensions/uberfire-security/uberfire-security-management/uberfire-widgets-security-management/src/main/java/org/uberfire/ext/security/management/client/widgets/management/explorer/EntitiesExplorerViewImpl.java [hunk 2]
```diff
@@ -64,10 +72,10 @@
     TextBox searchBox;
     
     @UiField
-    InputGroupAddon searchButton;
+    Button searchButton;
```
Comment (manstis) [inline uberfire-extensions/uberfire-security/uberfire-security-management/uberfire-widgets-security-management/src/main/java/org/uberfire/ext/security/management/client/widgets/management/explorer/EntitiesExplorerViewImpl.java]: If you're changing these to ```Button``` did you want to update the click [handler](https://github.com/cristianonicolai/uberfire/blob/ae7bd47f49aaf633494c166e41eb18c6dc7ad228/uberfire-extensions/uberfire-security/uberfire-security-management/uberfire-widgets-security-management/src/main/java/org/uberfire/ext/security/management/client/widgets/management/explorer/EntitiesExplorerViewImpl.java#L113) code to use ```addClickHandler(..)``` too?

**Comment by manstis** [comment]
+1 to merge (with others) - comment about click handler can be ignored if you like.

**Comment by ederign** [comment]
@csadilek could you please make the synced merge?

File: uberfire-extensions/uberfire-security/uberfire-security-management/uberfire-widgets-security-management/src/main/java/org/uberfire/ext/security/management/client/widgets/management/explorer/EntitiesExplorerViewImpl.java (hunk 2 above)
Comment (cristianonicolai) [inline uberfire-extensions/uberfire-security/uberfire-security-management/uberfire-widgets-security-management/src/main/java/org/uberfire/ext/security/management/client/widgets/management/explorer/EntitiesExplorerViewImpl.java]: @manstis sure, I can update that as well.

**Pull request merged**